source/how-govuk-sign-in-works.html.md.erb

```diff
@@ -1,68 +1,46 @@
+1. Your service asks the user to sign in or create an account.
+1. If your service needs confidence your user is who they say they are, GOV.UK Sign In will request proof of identity.
+1. GOV.UK Sign In collects evidence of the user’s identity.
+1. GOV.UK Sign In provides information about your user.
 
-<%= image_tag "generating-tokens.svg", { :alt => '' } %>
+You can read [guidance about cookies on GOV.UK](https://www.gov.uk/help/cookie-details) if you want to learn more about cookies for GOV.UK Accounts.
 
+To understand the technical flow, for example the endpoints, requests and tokens, there's a more detailed technical diagram you can use.
 
-1. After getting your user’s consent, GOV.UK Sign In redirects your user back to your service, along with an authorisation code tagged onto the end of the URL.
-1. Your service calls the `/token` endpoint with the authorisation code to get an ID token and an access token.
-1. GOV.UK Sign In returns an ID token and an access token to your service.
-1. Optionally, your service can use the access token to make a call to the `/userinfo` endpoint to access additional attributes if the user has agreed to share these.
 
-## Understand the environments GOV.UK Sign In uses
+## Understand the technical flow GOV.UK Sign In uses
 
-GOV.UK Sign In uses 2 different environments:
+<%= image_tag "technical-flow-diagram.svg", { :alt => '' } %>
 
-* an integration environment, which contains sample users and data for you to test your service’s integration with GOV.UK Sign In
-* a production environment, which is the live environment for real users to access and use your service’s integration with GOV.UK Sign In
+1. To make an authorisation request, your service sends the user to the `/authorize` endpoint.
+1. The user logs in (or creates an account if they do not have one) and proves their identity if your service needs them to. GOV.UK Sign In lets your user know how their data will be shared with your service.
+1. GOV.UK Sign In returns an authorisation code.
+1. Your service makes a [token request](/integrate-with-integration-environment/integrate-with-code-flow/#make-a-token-request) to the `/token` endpoint and includes the authorisation code in the request.
+1. Your service receives an ID token, an access token and a refresh token in the response.
+1. Your service makes a request to the `/userinfo` endpoint to [retrieve user information](/integrate-with-integration-environment/integrate-with-code-flow/#retrieve-user-information). You can read more about [choosing which user attributes your service can request](/integrate-with-integration-environment/choose-which-user-attributes-your-service-can-request/).
+1. Your service receives a response containing user attributes.
 
 You can [get started with integrating with GOV.UK Sign In's integration environment](/integrate-with-integration-environment/).
 
+
 <%= partial "partials/links" %>
```

Comment:
Are there any partial links in this content? I cannot see any.

Reply:
no but it's standard to have this across all pages, in case we do use a partial link in a next evolution of this page

Reply:
got it